roles/confluent.common/tasks/redhat.yml

```diff
@@ -35,6 +35,10 @@
   yum:
     name: "{{redhat_java_package_name}}"
     state: present
+  register: java_install_result
```

Comment:
should also have reties on
- jar download and cli downloads
- the java install on debian/ubuntu as well